spec/rails_helper.rb

```diff
@@ -59,4 +59,6 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
   config.include FactoryBot::Syntax::Methods
+
+  include(ApplicationHelper)
```

Comment:
includeの括弧は基本省略して書きますね！
```suggestion
  include ApplicationHelper
```

Reply:
fda07f4dad370f97c89c20de7cc45ed32beece1b 修正しました。